### ISSUE TITLE
Default filter option is set to not approved instead of all on requests page

### ISSUE THREAD

**Issue by chrisausbun** [tidusjar/Ombi#719]
Plex Requests.Net Version: Development 1.10.1020

Operating System: Windows 10 Home

Reproduction Steps:

Whenever you navigate to the requests page the default filter option is set to only show not approved instead of all, when I set the default to all and I switch back and fourth between requests and issues, it changes back to not approved. This issue is very minimal as the new 1.10.1020 seemed to fix the issue of having to set the filter option to all before it would show anything at all everytime. I've listed some screenshots below.

![untitled](https://cloud.githubusercontent.com/assets/22944072/20849981/018abba2-b88e-11e6-97e8-67733ea6a37e.png)

If any logs will help, please let me know and I'll post them to pastebin and post the link.

**Comment by tidusjar** [comment]
The change was to make the default to Not approved for the admin, for regular users its set to all. 

Currently we will not save your default if you change it, it will only filter what is on screen. 

I guess you are expecting it to keep your filter when you change it?

**Comment by chrisausbun** [comment]
I did just notice that while I was signed in as admin so as long the regular users see all when they first click it, I'm happy. And, yeah I was kinda expecting it to save my all filter I choose when switching back and fourth between the different pages and when signing out then back in. But like I said, I'm not too concerned with that since all regular users should be good. The most important one to me would be the newsletter issues I reported. I received a lot of compliments from my users when they received that newsletter for the first time a few versions back.

 Thank you very much for looking into these issues.

On Dec 2, 2016, at 3:55 PM, Jamie <notifications@github.com<mailto:notifications@github.com>> wrote:


The change was to make the default to Not approved for the admin, for regular users its set to all.

Currently we will not save your default if you change it, it will only filter what is on screen.

I guess you are expecting it to keep your filter when you change it?

-
